toonz/sources/toonz/sceneviewer.cpp

```diff
@@ -1172,21 +1174,6 @@ void SceneViewer::drawCameraStand() {
     }
     glScaled(m_dpiScale.x, m_dpiScale.y, 1);
 
-    TImageP image = tool->getImage(false);
-
-    TToonzImageP ti = image;
-    if (ti) {
-      TRect imgRect(0, 0, ti->getSize().lx - 1, ti->getSize().ly - 1);
-      TRectD bbox = ToonzImageUtils::convertRasterToWorld(imgRect, ti);
-
-      TPixel32 imgRectColor;
-      // draw black rectangle instead, if the BlackBG check is ON
-      if (ToonzCheck::instance()->getChecks() & ToonzCheck::eBlackBg)
-        imgRectColor = TPixel::Black;
-      else
-        imgRectColor = TPixel::White;
```

Comment:
Sorry but I think I misunderstood your proposal.
I thought that you were planning to just make the box color editable, instead of replacing the box by whole background.

We do need the box itself to recognize the border of the image.
Also the background must become black when the "Black BG" option is activated.

Can you please reintroduce the box for Toonz Raster and Raster levels, replacing `TPixel::White` of this line with `Preferences::instance()->getLevelEditorBgColor()`?

Reply:
Later in the discussion it was suggested (if I understood correctly) to just have a different background color for the Level Editor.  I felt it made sense and still allowed Studio Ghibli to have their white background.  The bounding box is still there for both Toonz Raster and Raster levels so you know where the image will be and can be drawn.

Sorry, but I don't understand the original intention of just having TR having a filled white box and why this doesn't work for them. I do not need an explanation other than this is just another case of how it has to be.

I will go back to my original proposal and make just the box color changeable.

Reply:
> Sorry, but I don't understand the original intention of just having TR having a filled white box and why this doesn't work for them.

The reason why I inserted the white box in Ghibli edition was to keep consistency with their conventional Ink&Paint software, Toonz4.6. IMO the white background has its roots in that the drawing on white paper was put under celluloid on Ink&Paint work.
I implemented only for Toonz Raster because Ghibli does not use other level types for animation levels.

It is true that OT shows raster bounding box, but I think for them it is more of visual cue for grasping the placement of each level on compositing scenes. On ink & painting they'd prefer to have more prominent boundary than light-gray and thin border line. The filled box was one of the simple solutions for us.
Everyone has different opinions on UI... I'm sorry to trouble you.

Reply:
What if the bounding box boundary was made more prominent?

Reply:
Hmm, of course I must confirm to my colleagues.., but I think the prominent border line may become noisy on compositing and if we make the line thicker we need to be careful to prevent the line from hiding the edge pixels of the image.

Reply:
@shun-iwasawa 
I find the virtue of consistency in @manongjohn solution, while I understand what you are saying about the needs and habits of Ghibli's staff.

What if one of the following ideas was implemented for Toonz Raster and Raster levels, so that Ghibli's staff could still have a stronger image size cue, while preserving the same background for all level types?:
- The **Level Editor BG Color** could be used as the background *inside* the limits of the level image, and the **Viewer BG Color** used *outside* of those limits.
- Alternatively, a new "image size masking" feature (with a % fade value), could be implemented, that would allow to darken the outside of the image limits in Level Edit mode, so there would be much more contrast between inside and outside areas of the level image.

Reply:
@gab3d 

> - The Level Editor BG Color could be used as the background inside the limits of the level image, and the Viewer BG Color used outside of those limits.

Please correct me if I am wrong. It seems that the first option is exactly talking about "the box" which is currently implemented (by me) as white-only on Toonz-Raster-only. Right?
If so, only the first option would be sufficient for us.

Reply:
yep, I think both options would achieve the goal of more clearly define the limits of the level image, but are somewhat different one from another.
The first one is very similar to what you had implemented, other than the Level Editor BG Color is now customizable.
The second one attains the contrast in a different way (by fading a mask over BG color) and has the benefit of not depending on the Viewer BG Color at all to produce the contrast, so if a user decides to change the Viwer BG color to the same color as the Level Editor BG Color, the image limits would still be there correctly.